Plugins/MeshSetPlugin/FrostyMeshSetEditor.cs

```diff
@@ -1694,6 +1718,26 @@ private List<AxisAlignedBox> ToAxisAlignedBoundingBoxes(List<BoundingBox> inBoun
             return retVal;
         }
 
+        private AxisAlignedBox ToAxisAlignedBoundingBox(BoundingBox inBoundingBox)
+        {
+            AxisAlignedBox retVal = new AxisAlignedBox();
+
+            Vec3 min = new Vec3();
+            min.x = inBoundingBox.Minimum.X;
+            min.y = inBoundingBox.Minimum.Y;
+            min.z = inBoundingBox.Minimum.Z;
+
+            Vec3 max = new Vec3();
+            max.x = inBoundingBox.Maximum.X;
+            max.y = inBoundingBox.Maximum.Y;
+            max.z = inBoundingBox.Maximum.Z;
+
+            retVal.min = min; 
+            retVal.max = max;
+
+            return retVal;
```

Comment:
u could also use just return new AxisAlignedBox() { min = new Vec3() { x = ... }, max = ... }, but doesnt really matter that much